src/Altinn.Common/Altinn.EFormidlingClient/Altinn.EFormidlingClient/README.md

```diff
@@ -0,0 +1,79 @@
+﻿
+## Getting Started
+
+These instructions will get help you setup eFormidlingClient package in your solution as well as how to test it.
+
+### Prerequisites
+
+The client package is written in C# and targets .Net Standard 2.1. The API solution which the package communicates with, i.e. the integration point, is written in Java. The integration point is possible to run locally with mock settings for testing purposes, requiring minimum Java 8 SDK. More on this in the test section.
+
+#### eFormidlingClient package
+1. [.NET Standard 2.1](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)
+2. Code editor of your choice
+3. Newest [Git](https://git-scm.com/downloads)
+4. Solution is cloned
+
+#### Integration Point
```

Comment:
This integration point isn't really a prerequisite tho.. They would do fine only using the mock. 
I think this can be written with the user's of Altinn Studio in mind rather than a broader audience. We could make it more general in the future.